### PR TITLE
A few server mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 1]
```diff
@@ -92,17 +100,38 @@ CLASS net/minecraft/server/MinecraftServer
...
 	METHOD a isSpawnProtected (Lbhh;Lev;Lavx;)Z
 	METHOD a getWorld (Lbyd;)Lvg;
+	METHOD a kickNonWhitelistedPlayers (Lcd;)V
+		ARG 1 source
+	METHOD a initScoreboard (Lcnz;)V
+		ARG 1 persistentStateManager
+	METHOD a setToDebugWorldProperties (Lcoa;)V
+		ARG 1 properties
 	METHOD a createWorlds (Lcob;Lcoa;Lbhl;Lvp;)V
+		ARG 2 perperties
```
Comment (Gegy) [inline mappings/net/minecraft/server/MinecraftServer.mapping]: I imagine this is a typo?

File: mappings/net/minecraft/server/command/TeleportCommand.mapping [hunk 2]
```diff
@@ -1,4 +1,11 @@
 CLASS tr net/minecraft/server/command/TeleportCommand
+	CLASS net/minecraft/server/command/TeleportCommand$a LookTarget
```
Comment (Gegy) [inline mappings/net/minecraft/server/command/TeleportCommand.mapping]: Knit bug

File: mappings/net/minecraft/server/MinecraftServer.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/server/MinecraftServer.mapping]: Fixed

File: mappings/net/minecraft/server/command/TeleportCommand.mapping (hunk 2 above)
Comment (Runemoro) [reply mappings/net/minecraft/server/command/TeleportCommand.mapping]: Fixed

**Review by liach — APPROVED** [review]

**Review by NeunEinser — APPROVED** [review]

**Review by asiekierka — APPROVED** [review]

**Pull request merged**